pkg/cri/server/container_create_windows.go

```diff
@@ -118,6 +120,7 @@ func (c *criService) containerSpec(
 		customopts.WithAnnotation(annotations.SandboxName, sandboxConfig.GetMetadata().GetName()),
 		customopts.WithAnnotation(annotations.ContainerName, containerName),
 		customopts.WithAnnotation(annotations.ImageName, imageName),
+		customopts.WithAnnotation(annotations.WindowsHostProcess, strconv.FormatBool(sandboxConfig.GetWindows().GetSecurityContext().GetHostProcess())),
```

Comment:
Do we also need a check like [this](https://github.com/containerd/containerd/blob/main/pkg/cri/server/container_create_linux.go#L216-L219) to disallow creating a HostProcess container in a non-HostProcess pod?

Reply:
Well I saw it was merged already :) Might be worth addressing in a follow-up PR though.

Reply:
We actually handle this in the shim right now, but wouldnt hurt here also if we remove that

Reply:
This is the right place for it